builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -87,12 +89,34 @@ node() {
     }
   }
 
+  accountDetails = utilModule.getAccountInfo(config);
+  accountDetailsPrimary = utilModule.getAccountInfoPrimary();
+
   if (!config) {
     error "Failed to fetch service metadata from catalog"
   }
 
+  if(!config.region) {
+    config.region = configLoader.AWS.DEFAULTS.REGION
+  }
+
+  if(!config.accountId) {
+    config.accountId = configLoader.AWS.DEFAULTS.ACCOUNTID
+  }
+
+  if(!config.provider) {
+    config.provider = configLoader.JAZZ.DEFAULTS.PROVIDER
+  }
+
+  def jazz_prod_api_id = getApiIDName(accountDetailsPrimary)
+  g_base_url = "https://${jazz_prod_api_id}.execute-api.${configLoader.AWS.DEFAULTS.REGION}.amazonaws.com/prod"
```

Comment:
Please see the below comment regarding this

Reply:
removed